src/preact.d.ts

```diff
@@ -96,7 +99,7 @@ declare namespace preact {
 
 		forceUpdate(callback?: () => void): void;
 
-		abstract render(props?: RenderableProps<P>, state?: Readonly<S>, context?: any): JSX.Element | null;
+		abstract render(props?: ClassRenderableProps<P>, state?: Readonly<S>, context?: any): JSX.Element | null;
```

Comment:
I'm not sure if this is correct. My understanding is that `html` tags can have `refs`, but not a component per se.

Reply:
I think Preact supports refs on class components ([example](https://codesandbox.io/s/52p18zyj4n)). Check the console output of that window. If this isn't supposed to be used however, I'll remove it. Just let me know.

Reply:
FYI, the linked codesandbox is just a copy of the relevant components from the preact.tsx test file to demonstrate usage of refs.

Reply:
You're right. Thanks for the follow up :)